spec/20-http_request_header_format.md

```diff
@@ -258,8 +258,8 @@ A simple example of a `list` with two `list-member`s might look like:
 
 ``` abnf
 list  = list-member 0*31( OWS "," OWS list-member )
-list-member = key "=" value
-list-member = OWS
+list-member =  key "=" value
+list-member =/ OWS
```

Comment:
Personally I'd find this definition clearer:
```suggestion
list-member = (key "=" value) / OWS
```
Maybe this is just because that's the syntax I'm more used to as I haven't seen `=/` before.

Reply:
WDYT @SergeyKanzhelev?

Reply:
it's cosmetic. I'm fine either way